docs/api/browser-window.md

```diff
@@ -1626,6 +1626,8 @@ Returns `Boolean` - Whether the menu bar is visible.
 * `options` Object (optional)
   * `visibleOnFullScreen` Boolean (optional) _macOS_ - Sets whether
     the window should be visible above fullscreen windows
+  * `skipTransformProcessType` Boolean (optional) _macOS_ - Sets whether
+    the process type should be transformed. Default is `false`
```

Comment:
While functionally true, this description doesn't seem really helpful to developers.  Can you reword to provide better insight to developers as to why you would want to set this?

Reply:
Sure! How does the updated description sound?